node-red-contrib-bot-message/bot-send-card.html

```diff
@@ -801,21 +835,47 @@
+                <label for="node-input-placeHolderformAdaptiveCard"><i class="fa fa-pencil-square-o"></i> Placeholder</label>
+                <input type="text" id="node-input-placeHolderformAdaptiveCard" style="width:80%;">
+                <input type="hidden" id="node-input-placeHolderformTypeAdaptiveCard">
+            </div>
+            <div class="form-row">
+                <label for="node-input-errorMsgformAdaptiveCard"><i class="fa fa-pencil-square-o"></i> Error Message</label>
+                <input type="text" id="node-input-errorMsgformAdaptiveCard" style="width:80%;">
+                <input type="hidden" id="node-input-errorMsgformTypeAdaptiveCard">
+            </div>
+            <div class="form-row">
+                <label for="node-input-submitLabelformAdaptiveCard"><i class="fa fa-pencil-square-o"></i> Submit Label</label>
+                <input type="text" id="node-input-submitLabelformAdaptiveCard" style="width:80%;">
+                <input type="hidden" id="node-input-submitLabelformTypeAdaptiveCard">
             </div>
         </section>
 
         <section class="section-type section-formGlobalAdaptiveCard">
             <div class="form-row">
-                <label for="node-input-titleAdaptiveCard"><i class="fa fa-pencil-square-o"></i> Title</label>
-                <input type="text" id="node-input-titleAdaptiveCard" style="width:80%;">
-                <input type="hidden" id="node-input-titleTypeAdaptiveCard">
+                <label for="node-input-titleformGlobalAdaptiveCard"><i class="fa fa-pencil-square-o"></i> Title</label>
+                <input type="text" id="node-input-titleformGlobalAdaptiveCard" style="width:80%;">
+                <input type="hidden" id="node-input-titleformGlobalTypeAdaptiveCard">
+            </div>
+            <div class="form-row">
+                <label for="node-input-placeHolderformGlobalAdaptiveCard"><i class="fa fa-pencil-square-o"></i> Placeholder</label>
+                <input type="text" id="node-input-placeHolderformGlobalAdaptiveCard" style="width:80%;">
+                <input type="hidden" id="node-input-placeHolderformGlobalTypeAdaptiveCard">
+            </div>
+            <div class="form-row">
+                <label for="node-input-errorMsgformGlobalAdaptiveCard"><i class="fa fa-pencil-square-o"></i> Error Message</label>
+                <input type="text" id="node-input-errorMsgformGlobalAdaptiveCard" style="width:80%;">
+                <input type="hidden" id="node-input-errorMsgformGlobalTypeAdaptiveCard">
+            </div>
+            <div class="form-row">
+                <label for="node-input-submitLabelformGlobalAdaptiveCard"><i class="fa fa-pencil-square-o"></i> Submit Label</label>
+                <input type="text" id="node-input-submitLabelformGlobalAdaptiveCard" style="width:80%;">
+                <input type="hidden" id="node-input-submitLabelTypeformGlobalAdaptiveCard">
             </div>
         </section>
 
```

Comment:
To add doc for 2 newly-added types of cad 

![image](https://user-images.githubusercontent.com/15153981/143842994-933a2731-80a5-4354-89af-aeab65d1011c.png)